### PR TITLE
Add cpg0016 listing to cellpainting-gallery

### PR DESCRIPTION
**Pull request by shntnu**

### CODE REVIEW

**Comment by shntnu** [comment]
@ErinWeisbart Could you please suggest what edits to make here? We can merge on Oct 31

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ E.g. the accession number of the first dataset is `cpg0000`.
 | cpg0004-lincs                            | 1,571 compounds across 6 doses in A549 cells                                                                                    | 4                   | idr0125                  |
 | cpg0012-wawer-bioactivecompoundprofiling | 30,000 compound dataset in U2OS cells                                                                                           | 1,2                 | idr0016                  |
 | cpg0015-heterogeneity                    | 2,200+ compounds and 200+ genes profiles in U2OS cells                                                                          | 8                   | idr0016,idr0036, idr0033 |
+| cpg0016-jump                    | 116,000+ compounds and 16+ genes (CRISPR knockout and overexpression) profiled in U2OS cells                                                                          | 9                   |  |
```
Comment (ErinWeisbart) [inline README.md:32]:
```suggestion
| cpg0016-jump                    | 116,000+ compounds and 15,000+ genes (CRISPR knockout and overexpression) profiled in U2OS cells   | 9                   |  |
```
definitely more than 16 genes :)

**Review by ErinWeisbart — APPROVED** [review]
Other than severely downplaying the genetic contribution, LGTM

File: README.md (hunk 1 above)
Comment (shntnu) [reply README.md:32]: HAHAHAHA That would be something!!!

**Comment by shntnu** [comment]
LOL!!

Thank you!

I'll merge this on Oct 31 https://github.com/orgs/jump-cellpainting/projects/6

**Comment by ErinWeisbart** [comment]
@shntnu FYI I just added in an `a` and `b` to more clearly distinguish between the two Chandrasekaran et al. 2022 publications.

**Pull request merged**